build.gradle

```diff
@@ -34,7 +34,7 @@ apply {
         download {
             src "https://raw.githubusercontent.com/web3j/build-tools/master/gradle/$buildScript/build.gradle"
             dest "$rootDir/gradle/$buildScript/build.gradle"
-            overwrite true
+            overwrite false
```

Comment:
This shouldn't have changed

Reply:
this will be changed back when web3j is released